src/WebJobs.Script/Config/RouteHandlingOptions.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using AuthLevel = Microsoft.Azure.WebJobs.Extensions.Http.AuthorizationLevel;
+
+namespace Microsoft.Azure.WebJobs.Script.Configuration
+{
+    /// <summary>
+    /// Defines the routing mode for HTTP requests.
+    /// </summary>
+    public enum RouteHandlingMode
+    {
+        /// <summary>
+        /// Routes are mapped based on httpTrigger functions defined in function metadata (default).
+        /// </summary>
+        Function,
+
+        /// <summary>
+        /// Creates a single catch-all route that handles all requests and proxies them to the custom handler.
+        /// </summary>
+        All
+    }
+
+    public class RouteHandlingOptions
```

Comment:
Since this is now in Custom handler section. We can simply add these properties [HttpWorkerOptions](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs) class. We don't need to create RouteHandlingOptions.